frontend/src/v5/ui/routes/viewer/groups/groupsList.component.tsx

```diff
@@ -41,7 +41,9 @@ const GroupTreeItem = ({ item, collapse, disabled }) => {
 
 export const GroupTree = ({ tree, collapse, disabled }) => (
 	<GroupsTreeList>
-		{tree.map((item) => (<GroupTreeItem item={item} collapse={collapse} disabled={disabled} />))}
+		{tree.map((item) => (
+			<GroupTreeItem item={item} collapse={collapse} disabled={disabled} key={item._id || item.name} />
```

Comment:
Unfortunately not, because of the group of groups
![image](https://user-images.githubusercontent.com/43547703/182586250-78dbdbd2-e666-4887-af41-cf07edede160.png)

Nor is just the name
> ![image](https://user-images.githubusercontent.com/43547703/182586564-f54ca057-0d1d-4961-ac31-699532df1e8e.png)
> **Example** of a condition that triggers an error using just the name as the key

Reply:
but id is not the name 🤔

Reply:
The issue is that a group of group comes without an id, so the key is empty and react complaints about that